### ISSUE TITLE
Improve readability

### ISSUE THREAD

**Issue by bviktor** [akiko-pusu/redmine_issue_badge#62]
A picture is worth a thousand words. This is what it looks like with default theme:

![image](https://user-images.githubusercontent.com/5798552/27658702-2426a682-5c51-11e7-891b-5016d3a3249e.png)

This is alternate:

![image](https://user-images.githubusercontent.com/5798552/27658823-9c90d638-5c51-11e7-88c3-5b8e9d9e9775.png)

Classic:

![image](https://user-images.githubusercontent.com/5798552/27658906-f195cb5c-5c51-11e7-9781-026af3396d5f.png)

You get the idea. It's way too small. The red-on-white contrast doesn't help either. I'm not quite sure what the proper fix would be. After some quick fiddling:

![image](https://user-images.githubusercontent.com/5798552/27659522-1abe0e2a-5c54-11e7-9dc6-a11f6d71888a.png)

Basically just:

~~~
padding: 3px 5px 3px 3px;
font-size: 0.8em;
color: #fff;
background-color: #ee2630;
~~~

I **don't** this should be the final code, I'm just trying to demonstrate what I imagine would be a more readable look. What do you guys think?

**Comment by akiko-pusu** [comment]
Thanks for your feedback.
But I can not imagine what color that everyone likes....
In case normal (PC) screen, badge color is based on red, on the other hands, responsive mode is based on green.
* Ref. http://www.redmine.org/plugins/redmine_issue_badge

As one solution, you can customize your CSS with using View Customize plugin.
This plugin is quite helpful if you want to change CSS / JavsScript through browser setting.
https://github.com/onozaty/redmine-view-customize

Another solution is be able to change / customize the coloring on plugin’s setting screen.

**Comment by bviktor** [comment]
Yeah, it's a tricky one, there's no "one size fits all" solution to visual stuff. This particularly applies to this case foz you can use custom themes in Redmine...

I like the idea of exposing some CSS attributes on the plugin's settings screen in administration. Would `font-size`, `color` and `background-color` be acceptable for you?

**Comment by akiko-pusu** [comment]
@bviktor 
Thanks again. Usually I check the style with Chrome, but it seems Chrome is not respecting rem / em font size. So that I could not find the badge font size was too small and hard to read except Chrome.
(Sorry I confirmed the situation with using Safari and Firefox.)

Now I fix css to follow its font-size with Redmine’s header, on commit: 	9a6ac40.
Thanks your pointed out!

On the other hand, this feature is nice to me. I’m going to try next release.
> I like the idea of exposing some CSS attributes on the plugin's settings screen in administration.  Would font-size, color and background-color be acceptable for you?

**Comment by bviktor** [comment]
Thanks a lot, indeed it got much better!
